extensions/interactions/MathExpressionInput/directives/oppia-interactive-math-expression-input.directive.ts

```diff
@@ -206,7 +206,15 @@ angular.module('oppia').directive('oppiaInteractiveMathExpressionInput', [
                 }
               }
             });
+            // @ts-ignore
+            // Ts Ignore is used here because actually Guppy is a class
+            // constructor having some properties. Ts doesn't support these
+            // kind of objects.
             if (angular.equals(Guppy.Symbols.symbols, {})) {
+              // @ts-ignore
+              // Ts Ignore is used here because actually Guppy is a class
+              // constructor having some properties. Ts doesn't support these
+              // kind of objects.
```

Comment:
Could you explain this a bit more, maybe add links to some explanation online?

Reply:
Ok. Let there be a class `SampleClass`. Now we can create an object belonging to this class like this.
```typescript
const sampleObject = new SampleClass();
```
but there are no properties of a constructor class like.
```typescript
const a = SampleClass.property;
```
Guppy has some properties like these like `init`. So this had to be done.

Reply:
So let us rewrite it to something like: `Type system does not support constructor properties (Guppy.init) thus we need ts-ignore it.`

Reply:
Done